pyproject.toml

```diff
@@ -94,7 +94,7 @@ mysql = ["pymysql", "pytest-mysql"]
 litellm = ["litellm"]
 neo4j = ["neo4j"]
 sciphi = ["agent-search"]
-
+metaphor = ["metaphor-python"]
```

Comment:
There needs to be a dependency on metaphor marked with optional=True

Reply:
added